tools/jit/gen_jit_dispatch.py

```diff
@@ -441,7 +441,9 @@ def format_arg(arg):
     if len(decl['returns']) == 1:
         ret_list = jit_type_of(decl['returns'][0])
     else:
-        ret_list = '({})'.format(', '.join(jit_type_of(r) for r in decl['returns']))
+        def type_maybe_field(r):
+            return '{} {}'.format(jit_type_of(r), r['field_name']) if 'field_name' in r else jit_type_of(r)
+        ret_list = '({})'.format(', '.join(type_maybe_field(r) for r in decl['returns']))
```

Comment:
What does this change do?

Reply:
If return field named are defined in `native_functions.yaml`, then it will return `-> (Tensor values, Tensor indices)` instead of `-> (Tensor, Tensor)`